packages/hop-node/src/watchers/classes/BaseWatcher.ts

```diff
@@ -78,7 +78,7 @@ class BaseWatcher extends EventEmitter implements IBaseWatcher {
     }
     if (config.bridgeContract != null) {
       if (this.isL1) {
-        this.bridge = new L1Bridge(config.bridgeContract as L1BridgeContract | L1ERC20BridgeContract)
+        this.bridge = new L1Bridge(config.bridgeContract as L1BridgeContract)
```

Comment:
Should `L1ERC20BridgeContract` be removed from this entire file? Looks like it is just the import and the config declaration as well.

Reply:
Agreed

Resolved https://github.com/hop-protocol/hop/pull/384/commits/91ffb6b5c6181e25b417eec50bb6f79e7a7cd6ed